src/aura/BGE_DataImportBatchEntry/BGE_DataImportBatchEntryHelper.js

```diff
@@ -481,9 +485,13 @@
                         header: modalHeader,
                         footer: modalFooter,
                         showCloseButton: true,
-                        cssClass: 'slds-modal_large'
+                        cssClass: 'slds-modal_large',
+                        closeCallback: function() {
+                            component.set('v.modalOpen', false);
+                        }
                     })
                 } else {
+                    component.set('v.modalOpen', false);
```

Comment:
What do you think about doing the same thing here and using a named function in both places?